xref-token/src/owner.rs

```diff
@@ -15,11 +15,26 @@ impl Contract {
         self.owner_id.clone()
     }
 
-    pub fn modify_reward_per_sec(&mut self, reward_per_sec: U128) {
+    pub fn modify_reward_per_sec(&mut self, reward_per_sec: U128, distribute_before_change: bool) {
```

Comment:
why do we need a flag here? isn't distribute reward always required when rps changes?

Reply:
distribute_before_change::true => those undistribute till call time would distribute with old rps;
distribute_before_change::false => those undistribute till call time would distribute with new rps;
We provide possibility at contract level and caller makes choice.